tools/jlm-hls/jlm-hls.cpp

```diff
@@ -102,6 +102,7 @@ main(int argc, char ** argv)
     }
 
     jlm::hls::VerilatorHarnessHLS vhls;
+    vhls.SetIncludeFileName(commandLineOptions.OutputFiles_.base());
```

Comment:
Hand in the proper include filepath instead of just a string `commandLineOptions.OutputFiles_ ¨".h"`. (You might need to wrap it in `util::filepath()`)

Reply:
Also, why is this called `OutputFiles_` when it only contains a single file?

Reply:
Changed to outputVerilogFile